cards/src/main/java/com/decathlon/vitamin/compose/cards/VitaminCardSizes.kt

```diff
@@ -0,0 +1,25 @@
+package com.decathlon.vitamin.compose.cards
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+@Immutable
+data class CardSizes(
+    val elevation: Dp
+)
+
+object VitaminCardSizes {
+    @Composable
+    fun medium(
```

Comment:
I would have called it default() because this is the only size we offer for now

Reply:
"When you need to create an object access for sizes configurations but you have only one variant,
it should be named `medium`, excepted if the design specification defines the size in the Figma
project." dans le fichier convention c'est marqué ça.

Reply:
yes you're right my bad ;)